server/security.go

```diff
@@ -128,6 +128,7 @@ func (a *ApplicationHandler) CreatePersonalAPIKey(w http.ResponseWriter, r *http
 // @Failure 400,401,500 {object} util.ServerResponse{data=Stub}
 // @Security ApiKeyAuth
 // @Router /ui/organisations/{orgID}/groups/{groupID}/apps/{appID}/keys [post]
+func _() {}
```

Comment:
Why's this function empty?

Reply:
For `swag` to generate docs for annotations, the annotations must precede a function declaration. This is actually a bug that was fixed in this pr. There were some annotations that never got generated previously.

Reply:
I see, ok